tests/post_training/test_quantize_conformance.py

```diff
@@ -132,6 +133,12 @@ def validate_accuracy(model_path, val_loader):
     references = [-1] * dataset_size
 
     core = ov.Core()
+
+    if os.environ.get("CPU_THREADS_NUM"):
+        # Set CPU_THREADS_NUM for OpenVINO inference
+        cpu_threads_num = os.environ.get("CPU_THREADS_NUM")
+        core.set_property("CPU", properties={"CPU_THREADS_NUM": str(cpu_threads_num)})
```

Comment:
Is it works properly and as expected? Don't we need to use other parameters like `PERFORMANCE_HINT`?

Reply:
Yes, works as expected. The problem that in ci `cpu_limit` implemented via throttling (some more info https://komodor.com/learn/kubernetes-cpu-limits-throttling/), but number of available core is not changed. 

For example for `num_cores=32` and `cpu_limit=8`:
`throttling_factor = 8 / 32 = 0.25`
As result runs 32 threads but that used only 0.25 for each cores.   
In case of running 4 jobs on one host, this can leads to performance degradation.
`CPU_THREADS_NUM` force openivino run only limited threads, as result all thread will used 100% of each core and avoid throttling. 

Looks like it's enough, no need use `PERFORMANCE_HINT`.

Reply:
I hope that it was tested in the CI instance and wouldn't be the reason for the resource leaks and queuing of the parallel jobs on the same instance.